core/shared/src/main/scala/org/typelevel/log4cats/extras/DeferredLogger.scala

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats.extras
+
+import cats.data.Chain
+import cats.effect.kernel.Resource.ExitCase
+import cats.effect.kernel.{Concurrent, Ref, Resource}
+import cats.syntax.all.*
```

Comment:
Let's keep coherency with the old-fashioned but widely adopted imports across the Log4cats
```suggestion
import cats.syntax.all._
```